feature/rds/auth/connect.go

```diff
@@ -4,33 +4,26 @@ import (
 	"context"
 	"fmt"
 	"net/http"
-	"net/url"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/aws/signer/v4"
-	"github.com/aws/aws-sdk-go-v2/internal/sdk"
 )
 
 const (
-	rdsAuthTokenID    = "rds-db"
-	rdsClusterTokenID = "dsql"
-	emptyPayloadHash  = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
-	userAction        = "DbConnect"
-	adminUserAction   = "DbConnectAdmin"
+	signingID        = "rds-db"
```

Comment:
Are we not going to remove this? why keep feature/rds/auth? if its going to be empty?